### DISCUSSION TITLE
Add files from sub-directory to archive root

### DISCUSSION THREAD

**Discussion by abemedia** [goreleaser/goreleaser#1475] [Ideas, Issue Triage, and General discussions]
I need to include DLLs for my windows builds and have different DLL files for 32 & 64 bit versions of the application. They have the same name so I can't keep them in the project root, however they need to be in the same directory as the binary within the archive. 

Is there any way of specifying a file from a sub-directory, yet having it within the archive root?

**Comment by caarlos0** [comment]
right now there isn't, no...
